### PR TITLE
chore(linting): Setup linting

### PR DESCRIPTION
**Pull request by amercier**
- [x] Add Prettier
- [x] Add Prettier to CI
- [x] Fix Prettier errors
- [x] Switch to [eslint-config-airbnb](https://www.npmjs.com/package/eslint-config-airbnb)
- [x] Fix ESLint errors

### CODE REVIEW

File: .prettierrc.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/**
+ * Prettier configuration file.
+ * For more details read https://prettier.io/docs/en/options.html
+ */
+module.exports = {
+  semi: false,
```
Comment (clementprevot) [inline .prettierrc.js]: :cry: :sob:

File: .prettierrc.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  singleQuote: true,
+  trailingComma: 'all',
+}
```
Comment (clementprevot) [inline .prettierrc.js]: Could we also use 4 tabs instead of 2. I like air and space in the code

File: .prettierrc.js (hunk 1 above)
Comment (amercier) [reply .prettierrc.js]: I personally prefer with semicolons, but let's keep it this ways for two reasons:
- Prettier is designed for 80 chars in mind, adding `;` on every line would force more line to split
- We already use `semi: false` on https://github.com/BackMarket/front, let's not diverge from other repos for preference reasons

File: .prettierrc.js (hunk 2 above)
Comment (amercier) [reply .prettierrc.js]: I disagree with this, for two reasons:
- Prettier is designed for 80 chars in mind, multiplying spacing x2 would force a lot more line to split
- Consensus is 2 spaces in JS community in general

**Comment by clementprevot** [comment]
We could also have set prettier to 120 chars like any modern project...

**Pull request merged**